### PR TITLE
Enable repository quotas

### PR DESCRIPTION
**Pull request by RyanWBloom**
Created an optional feature to collect quota metrics for those who have enabled this plugin:
https://github.com/jfrog/artifactory-user-plugins/tree/master/storage/repoQuota

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,12 +128,14 @@ Flags:
...
 | `artifactory.ssl-verify`<br/>`ARTI_SSL_VERIFY` | No | `true` | Flag that enables SSL certificate verification for the scrape URI. |
 | `artifactory.timeout`<br/>`ARTI_TIMEOUT` | No | `5s` | Timeout for trying to get stats from JFrog Artifactory. |
+| `artifactory.repo-quota-enable`<br/> `ARTI_REPO_QUOTA_ENABLE` | No  | `true`  | Enables the quota monitoring. |
 | `log.level` | No | `info` | Only log messages with the given severity or above. One of: [debug, info, warn, error]. |
 | `log.format` | No | `logfmt` | Output format of log messages. One of: [logfmt, json]. |
 | `ARTI_USERNAME` | *No | | User to access Artifactory |
 | `ARTI_PASSWORD` | *No | | Password of the user accessing the Artifactory |
 | `ARTI_ACCESS_TOKEN` | *No | | Access token for accessing the Artifactory |
 
+
```
Comment (jeremyspykerman) [inline README.md]: Do we need this empty space?

File: README.md [hunk 2]
```diff
@@ -140,6 +144,8 @@ Some metrics are not available with Artifactory OSS license. The exporter return
 
 | Metric | Description | Labels | OSS support |
 | ------ | ----------- | ------ | ------ |
+| artifactory_storage_repo_quota_used_percent | Percentage of quota used |`version`, `revision`, `branch`, `goversion` | ? |
```
Comment (jeremyspykerman) [inline README.md]: The list of labels is not correct. You can confirm the list of labels by looking at the metrics we had from the running system before.

File: README.md [hunk 3]
```diff
@@ -140,6 +144,8 @@ Some metrics are not available with Artifactory OSS license. The exporter return
...
+| artifactory_storage_repo_quota_bytes  | Artifactory quota in bytes |`version`, `revision`, `branch`, `goversion` | ? |
```
Comment (jeremyspykerman) [inline README.md]: The list of labels is not correct. You can confirm the list of labels by looking at the metrics we had from the running system before.

File: README.md [hunk 4]
```diff
@@ -126,12 +128,14 @@ Flags:
 | `artifactory.scrape-uri`<br/>`ARTI_SCRAPE_URI` | No | `http://localhost:8081/artifactory` | URI on which to scrape JFrog Artifactory. |
 | `artifactory.ssl-verify`<br/>`ARTI_SSL_VERIFY` | No | `true` | Flag that enables SSL certificate verification for the scrape URI. |
 | `artifactory.timeout`<br/>`ARTI_TIMEOUT` | No | `5s` | Timeout for trying to get stats from JFrog Artifactory. |
+| `artifactory.repo-quota-enable`<br/> `ARTI_REPO_QUOTA_ENABLE` | No  | `true`  | Enables the quota monitoring. |
```
Comment (jeremyspykerman) [inline README.md]: I believe the default is `false`, not `true`

File: README.md (hunk 2 above)
Comment (jeremyspykerman) [reply README.md]: Additionally, we could use a TODO message or comment specifying that we need to confirm if it works with the OSS version of artifactory.

Comment (jeremyspykerman) [reply README.md]: also, we need a way to indicate these 2 metrics are only enabled if the quotas feature is enabled.

File: README.md [hunk 5]
```diff
@@ -135,11 +138,14 @@ Flags:
 * Either `ARTI_USERNAME` and `ARTI_PASSWORD` or `ARTI_ACCESS_TOKEN` environment variables has to be set.
 
 ### Metrics
+The quota metrics are only available if the feature to collect them is enabled.
 
 Some metrics are not available with Artifactory OSS license. The exporter returns the following metrics:
 
 | Metric | Description | Labels | OSS support |
 | ------ | ----------- | ------ | ------ |
+| artifactory_storage_repo_quota_used_percent | Percentage of quota used |`name`, `package_type`, `type` | ? Need to confirm it works with OSS  |
```
Comment (jeremyspykerman) [inline README.md]: I would recommend using this format:
`TODO: confirm it works with OSS`

**Review by jeremyspykerman — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (RyanWBloom) [reply README.md]: changed line 147

File: README.md (hunk 4 above)
Comment (RyanWBloom) [reply README.md]: changed from true to false

File: README.md (hunk 3 above)
Comment (RyanWBloom) [reply README.md]: found the labels from running the system and changed to correct labels

File: README.md (hunk 2 above)
Comment (RyanWBloom) [reply README.md]: changed the labels. added the TODO message and created a way to indicate that these two metrics are only enabled if the quota feature is enabled.

File: README.md (hunk 1 above)
Comment (RyanWBloom) [reply README.md]: empty space removed

**Review by jeremyspykerman — APPROVED** [review]

**Review by gtmtech — APPROVED** [review]

**Pull request merged**